src/rules/mergers/member-delimiter-style.ts

```diff
@@ -0,0 +1,19 @@
+import { RuleMerger } from "../merger";
+
+export const mergeNoMemberDelimiterStyle: RuleMerger = (existingOptions, newOptions) => {
+    if (existingOptions === undefined && newOptions === undefined) {
+        return [];
+    }
+
+    return [merge((existingOptions || [])[0] || {}, (newOptions || [])[0] || {})];
```

Comment:
The 100% coverage requirement is failing because one part of this line isn't hit. Without looking deeply at the report, my guess is it's that you're missing a test case for `old object config existing` (as in, only that one, and not the new config).